lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -108,13 +102,12 @@ public CheckExistingIdentityHandler() {
     @Override
     @Tracing
     @Logging(clearState = true)
-    public APIGatewayProxyResponseEvent handleRequest(
-            APIGatewayProxyRequestEvent event, Context context) {
+    protected JourneyResponse handleRequest(JourneyRequest event, Context context) {
         LogHelper.attachComponentIdToLogs();
 
         try {
-            String ipvSessionId = RequestHelper.getIpvSessionId(event);
-            String ipAddress = RequestHelper.getIpAddress(event);
+            String ipvSessionId = event.getIpvSessionId();
```

Comment:
One of the things RequestHelper does is attach the session id to the logs which we'd want to keep - https://github.com/alphagov/di-ipv-core-back/blob/9593f87b9ae6009227949d50dbb6f1babcb63b01/lib/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java#L114

Might be worth reusing StepFunctionsHelper which is used in the CRI return step function and adds the session id to the logs and does a null check